TriblerGUI/widgets/trustpage/js/radial_view/radial_inspector.js

```diff
@@ -0,0 +1,146 @@
+    self.config = Object.assign({}, defaults, options || {});
+
+    /**
+     * Select all <svg.link> objects in the SVG
+     */
+    self.select = function () {
+        return d3element;
+    };
+
+    /**
+     * Push the new data to the inspector and redraw
+     * @param {GraphData} newGraphData
+     */
+    self.onNewData = function (newGraphData) {
+        self.graphData = newGraphData;
+        self.displayNetworkInfo();
+    };
+
+    /**
+     * Display the network information
+     */
+    self.displayNetworkInfo = function () {
+        if (!self.graphData) return;
+
+        var data = self.graphData,
+            bindings = {
+                focusNodeName: self._getNodeIdentifier(data.focus_node),
+                nodeCount: data.nodes.length,
+                linkCount: data.links.length
+            };
+
+        self._setContents(
+            "Tribler Trust Network",
+            "Partial view around {focusNodeName}",
+            [
+                "Showing <strong>{nodeCount} users</strong>.",
+                "Click on users to show more."
+            ], bindings);
+    };
+
+    /**
+     * Display the node information
+     * @param {GraphNode} node
+     * @param {RadialNodes} radial_nodes - the radial nodes module, used for calculating the color
+     */
+    self.displayNodeInfo = function (node, radial_nodes) {
+        var balance = node.total_up - node.total_down,
+            bindings = {
+                formatted_upload: formatBytes(node.total_up),
+                formatted_download: formatBytes(node.total_down),
+                formatted_balance: (balance > 0 ? "+" : "") + formatBytes(balance),
+                peer_count: node.neighbors.length,
+                color: radial_nodes._calculateFill(node),
+            };
+
+        self._setContents(
+            self._getNodeIdentifier(node),
+            "Anonymous",
+            [
+                "Shared <strong>{formatted_upload}</strong> with {peer_count} users.",
+                "Consumed <strong>{formatted_download}</strong> from {peer_count} users.",
+                "<span class='badge' style='background:{color}'>Balance: <strong>{formatted_balance}</strong></span>.",
+            ], bindings);
+    };
+
+    /**
+     * Display the link information
+     * @param {GraphLink} link
+     */
+    self.displayLinkInfo = function (link) {
+        var bindings = {
+            source: self._getNodeIdentifier(link.source),
+            target: self._getNodeIdentifier(link.target),
+            formatted_up: formatBytes(link.amount_up),
+            formatted_down: formatBytes(link.amount_down)
+        };
+
+        self._setContents(
+            "Content exchanged",
+            "Between {source} and {target}",
+            [
+                "{source} shared <strong>{formatted_up}</strong> with {target}",
+                "{target} shared <strong>{formatted_down}</strong> with {source}",
+                "File content is unknown."
```

Comment:
We can change the text to "File contents are always hidden/secret/unknown/protected", but I do think we must address this issue here since this is where this question comes to mind.